src/argilla/client/feedback/dataset/remote/base.py

```diff
@@ -277,7 +277,9 @@ def pull(self) -> "FeedbackDataset":
             guidelines=self.guidelines,
             metadata_properties=self.metadata_properties,
         )
-        instance.add_records(
-            records=[record.to_local() for record in self._records],
-        )
+        records = [record.to_local() for record in self._records]
+
+        if records:
```

Comment:
Here we can just be more explicit:

```suggestion
        if len(records) > 0:
```